kahuna/public/js/mixpanel/mixpanel.js

```diff
@@ -47,6 +46,6 @@ mp.factory('mixpanel', ['$window', 'mixpanelEnabled', function($window, mixpanel
     return {
         init: init,
         track: track,
-        isEnabled: () => mixpanelEnabled
+        isEnabled: () => initialised
```

Comment:
It is still used [in one place](https://github.com/guardian/grid/blob/sc-mixpanel-token-in-api/kahuna/public/js/analytics/track.js#L87).